src/buffer/out/TextColor.h

```diff
@@ -45,6 +45,12 @@ enum class ColorType : BYTE
     IsRgb = 0x3
 };
 
+enum class ColorAlias : size_t
+{
+    DefaultForeground,
+    DefaultBackground
+};

```

Comment:
Do you expect that there will ever be more than these two?
Unrelated it'd be useful to define a "Max" or "Count" member of the enum here, so that the size of the alias array doesn't need to use the magic constant "2".

Reply:
Yeah, there definitely will be more than two at some point. Thanks for catching that. I've gone with `ENUM_COUNT`, because the only other example of this I could find was the [`ModifierKeyState`](https://github.com/microsoft/terminal/blob/fb597ed304ec6eef245405c9652e9b8a029b821f/src/types/inc/IInputEvent.hpp#L94) enum, and that's what was used there.